src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -628,7 +628,7 @@
       <_WasmAssembliesInternal Remove="@(_WasmAssembliesInternal)" />
 
       <_WasmAOTSearchPaths Include="$(MicrosoftNetCoreAppRuntimePackRidNativeDir)" />
-      <_WasmAOTSearchPaths Include="$(MicrosoftNetCoreAppRuntimePackRidDir)\lib\net8.0" />
+      <_WasmAOTSearchPaths Include="$(MicrosoftNetCoreAppRuntimePackRidDir)\lib\$(TargetFramework)" />
```

Comment:
You could use the parent directory from a known assembly, i.e. System.Private.CoreLib or mscorlib. I would probably pick the former as at some point in the distant future, the mscorlib shim assembly could go away.